### PR TITLE
Section team

### PR DESCRIPTION
**Pull request by tulupvica**

### CODE REVIEW

File: src/scss/components/expertise-card.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.expertise-card {
+
+  $heading-color: #413d4b;
```
Comment (ymazhar) [inline src/scss/components/expertise-card.scss]: Variables at the start of the file

File: src/scss/components/team-card.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.team-card {
+
+  $card-border-color: #89829c;
```
Comment (ymazhar) [inline src/scss/components/team-card.scss]: Variables at the start of the file

File: src/scss/sections/team.scss [hunk 3]
```diff
@@ -0,0 +1,75 @@
+.section-team {
+
+  $bg-gradient: linear-gradient(to right, rgba(9,5,47,0.85) 0%,rgba(49,29,94,0.85) 100%);
```
Comment (ymazhar) [inline src/scss/sections/team.scss]: Variables at the start of the file

File: src/scss/components/expertise-card.scss (hunk 1 above)
Comment (tulupvica) [reply src/scss/components/expertise-card.scss]: fixed

File: src/scss/components/team-card.scss (hunk 2 above)
Comment (tulupvica) [reply src/scss/components/team-card.scss]: fixed

File: src/scss/sections/team.scss (hunk 3 above)
Comment (tulupvica) [reply src/scss/sections/team.scss]: fixed

**Pull request merged**